guardian/support-service-lambdas
# Sf emails exporter   group emails by case number and save as json in s3

## What does this change?
Add logic to group the emails from Salesforce by Parent CaseNumber and save as json files to S3 with filename=casenumber

## How to test
run the lambda and verify that json files created in S3. 

Each Json file should:

- contain all emails relating to the Case

- be named by CaseNumber

## Coming Next
When writing to S3, rather than overwrite the file, append to it instead

### Previous PRs

[Create Basic Lambda - Sf emails to s3 exporter #1316](https://github.com/guardian/support-service-lambdas/pull/1316)

[Sf email exporter - authenticate with sf and get emails #1317](https://github.com/guardian/support-service-lambdas/pull/1317)
